### PR TITLE
Updated the Note for Android for outlook make it clearer.

### PR DESCRIPTION
**Pull request by j0rt3g4**
Updated Note for Issue #1399 
@JohanFreelancer9  this one probably needs some checks.

### CODE REVIEW

**Comment by j0rt3g4** [comment]
PR for issue #1399

File: Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/outlook-for-ios-and-android/manage-outlook-for-ios-and-android.md [hunk 1]
```diff
@@ -114,4 +114,6 @@ See [Mobile device mailbox policies in Exchange Online](../../clients-and-mobile
 Exchange administrators can initiate a remote device wipe against Outlook for iOS and Android. Upon receiving the remote wipe request, the app will remove the profile and all data associated with it.
 
 > [!NOTE]
-> Outlook for iOS and Android only supports the "Wipe Data" remote wipe command and does not support "Account Only Remote Wipe Device."
+> On Outlook for iOS and Android the following sentences are true:
+> * Supports the "Wipe Data" throught remote wipe command
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/outlook-for-ios-and-android/manage-outlook-for-ios-and-android.md]:
```suggestion
> * It supports "Wipe Data" through the remote wipe command.
```

File: Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/outlook-for-ios-and-android/manage-outlook-for-ios-and-android.md [hunk 2]
```diff
@@ -114,4 +114,6 @@ See [Mobile device mailbox policies in Exchange Online](../../clients-and-mobile
...
+> * Does not support "Account Only Remote Wipe Device" command.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/outlook-for-ios-and-android/manage-outlook-for-ios-and-android.md]:
```suggestion
> * It does not support the "Account Only Remote Wipe Device" command.
```

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by JohanFreelancer9** [comment]
@j0rt3g4 I notice that you have not committed my third suggestion?

**Comment by mattpennathe3rd** [comment]
@JohanFreelancer9 @j0rt3g4 Thanks for your help with this PR. I have reached out to a SME for clarification on this topic and will handle this one from here.

**Comment by mattpennathe3rd** [comment]
@j0rt3g4 @JohanFreelancer9 No further work on this PR, please. This article (plus another with similar content) requires significant rewriting and I am working on it with the SME for this area.

**Pull request closed without merging**